README.md

```diff
@@ -41,12 +41,12 @@ Here's how slack concepts are mapped to purple:
    * For bitlbee IRC connections, Slack channels are "chat channels" that can be added to your configuration with "`chat add <account id> #<channel>`"
 
 ### Available Commands
-- `/edit <new message>` will edit the last message you sent in a channel/chat to whatever you write. 
+- `/edit <new message>` will edit the last message you sent in a channel/chat to whatever you write.
 - `/delete` will delete the last message you sent in a channel/chat.
+- `/th <timestamp=HH:mm:ss> <message>` will reply to message with supplied timestamp in a thread.
+  Timestamp must be in 24-hour format.
```

Comment:
This depends on what you configure it to be. And it also depends on the locale. If you use the default, at least in Pidgin the timestamp is the same format as the timestamp which precedes messages.

What about simply: "The timestamp format is configurable in the account options."